skimage/filters/_median.py

```diff
@@ -3,12 +3,9 @@
 import numpy as np
 from scipy import ndimage as ndi
 
-from .._shared.utils import deprecate_kwarg
 from .rank import generic
 
 
-@deprecate_kwarg(kwarg_mapping={'selem': 'footprint'}, removed_version="1.0",
-                 deprecated_version="0.19")
```

Comment:
Before the 'Returns' section, shouldn't there be something like:
```py
    .. versionchanged:: 0.19
        Parameter ``selem`` has been replaced by ``footprint``.
```
? I think @jni would know...

Reply:
I'm pretty sure that we don't use the `.. versionchanged::`` directive consistently (yet). But I could go through and update each docstring if we deem it worth it to keep the notice around.